clients/traycer-cli/src/host/__tests__/lifecycle-probe-real-launchd.optional.integration.test.ts

```
@@ -0,0 +1,1433 @@
+    await runCommand("launchctl", ["bootout", "--wait", target]).catch(
+      () => undefined,
+    );
+    const remainsLoaded = await runCommand("launchctl", ["print", target])
+      .then(() => true)
+      .catch(() => false);
+    await rm(root, { recursive: true, force: true });
+    cleanupTargets = cleanupTargets.filter((candidate) => candidate !== target);
+    if (remainsLoaded) {
+      throw new Error(
+        `scoped M8 fallback remained loaded after teardown: ${target}`,
+      );
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)